build.sh

```diff
@@ -25,7 +25,7 @@ function usage()
 # Determine how many parallel jobs to use for make based on the number of cores
 unamestr="$(uname)"
 if [[ "$unamestr" == "Linux" ]]; then
-  PARALLEL=$(nproc)
+  PARALLEL=1
```

Comment:
without this, I'm getting the bad filedescriptor error from https://github.com/ray-project/ray/issues/3767. It doesn't impact build times too much and this will go away soon anyways.